app/services/spree/payment/epay_service.rb

```diff
@@ -0,0 +1,61 @@
+module Spree
+  module Payment
+    class EpayService
+      def call(order:, payment_method:)
+        @order = order
+        @payment_method = payment_method
+      end
+
+      def payment
+        response = connection.post(api_url, request_body)
+        response.body
+      end
+
+      private
+
+      # Setting to true will log requests and response to $stdout,
+      # should always be set to false when commiting code
+      DEBUG_MODE = false
```

Comment:
if this is related to printing errors, lets utilise debug level config(config.log_level) from config/development.rb or production.rb..
and act accordingly
(sorry for not putting down specific syntax implementation)